demo/src/main/java/com/novoda/demo/MainActivity.java

```diff
@@ -15,7 +15,7 @@
 public class MainActivity extends Activity {
 
     private static final String URI_VIDEO_MP4 = "http://yt-dash-mse-test.commondatastorage.googleapis.com/media/car-20120827-85.mp4";
-    private static final String URI_VIDEO_MPD = "https://storage.googleapis.com/content-samples/multi-audio/multi-audio.mpd";
+    private static final String URI_VIDEO_MPD = "https://storage.googleapis.com/content-samples/multi-audio/manifest.mpd";
```

Comment:
Link update so that we can test audio description without exposing assets from a different project.